website_sale_stock_control/models/product.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    inventory_availability = fields.Selection([
+        ('never', 'Allow buy products without control'),
+        ('always', 'Deny buy products without stock'),
+    ], string='Stock Control', default='never',
```

Comment:
`Website stock control`

Reply:
This field is inside "website" section

Reply:
Yeah, but when for example making an advanced search over the fields, you find the field "Stock Control" and you are not sure what does it means. In this case, I prefer to have as `string` "Website Stock Control", and then add as view level only "Stock Control". I know this is a general problem with usability across all Odoo, but let's contribute at least with this one. I write down in my TODO list to write a general guideline about this.